strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/LoggingManagementController.java

```diff
@@ -245,4 +262,115 @@ public ResponseEntity uploadLogbackConfiguration(HttpServletRequest request,
         }
     }
 
+    @ApiOperation(value = "Used to get log directory.",
+                  position = 0)
+    @ApiResponses(value = { @ApiResponse(code = 200,
+                                         message = "The log directory was retrieved successfully."),
+                            @ApiResponse(code = 400,
+                                         message = "Could not download log directory.") })
+    @GetMapping(value = { "/logs",
+                          "/logs/**" },
```

Comment:
Just for my better understanding, why are there two mappings for this?

Reply:
`/logs` is to capture any URI the user entered without any trailing forward slashes. I copied this approach from `BrowseStoragesController` that has the two mappings in line 38 and the method to perform the redirect in `BaseArtifactController` in line 176. I did both in one go. If I don't do that, navigating to `/logs` gives a 404 error code, unlike `/storages`.

Reply:
Okay, thanks for the explanation! :)

Reply:
I tried this out and it works only if `urlPath` is mentioned. If it isn't and the user wants to just access `/logs/` a 404 will be thrown. If I try out `urlPath:.*` as a 0 or more regex match, I get a MissingParamValue exception from Spring. This seems to only work with vanilla JAX-RS when I separately tried out a `hello/{world:.*}` pattern. This was my last resort to serve the home URL and any subseqent sub URLs. Also, Spring seems to limit regex capture to the next `/` character it encounters. So if someone enters `/logs/folder1/folder2`, `{urlPath:.+}` doesn't seem to capture anything after `folder1`